### PR TITLE
Freeze rules that will be 'published' to gitlab

### PR DESCRIPTION
**Pull request by minusworld**

### CODE REVIEW

File: gitlab/gitlab-bandit.yaml [hunk 1]
```diff
@@ -0,0 +1,965 @@
...
+    Running `socket.bind` to 0.0.0.0, ::, or empty string could unexpectedly
+    expose the server publicly as it binds to all available interfaces. Consider
+    instead getting correct address from an environment variable or
+    configuration file.
+  metadata:
+    category: security
+    cwe: 'CWE-200: Exposure of Sensitive Information to an Unauthorized Actor'
+    owasp: 'A6: Security Misconfiguration'
+    technology:
+    - pycryptodome
```
Comment (mschwager) [inline gitlab/gitlab-bandit.yaml:15]: A few of these don't look like `pycryptodome` rules :thinking:

File: gitlab/gitlab-eslint.yaml [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  message: |
+    Markup escaping disabled. This can be used with some template engines to escape
+    disabling of HTML entities, which can lead to XSS attacks.
+  metadata:
+    category: security
+    cwe: 'CWE-116: Improper Encoding or Escaping of Output'
+    owasp: 'A7: Cross-Site Scripting XSS'
+    source-rule-url: https://github.com/nodesecurity/eslint-plugin-security/blob/master/rules/detect-disable-mustache-escape.js
+    technology:
+    - wkhtmltopdf
```
Comment (mschwager) [inline gitlab/gitlab-eslint.yaml:15]: Similar issue here, most of these list `wkhtmltopdf`. This isn't a big deal for this particular PR, but looks like something we should fix.

**Comment by inkz** [comment]
@minusworld  @mschwager updates for meta here https://github.com/returntocorp/semgrep-rules/pull/1331
including fix for incorrect `technology` fields

**Comment by minusworld** [comment]
No lonver relevant

**Pull request closed without merging**